gpii/node_modules/flowManager/configs/gpii.flowManager.config.local.base.json5

```diff
@@ -6,6 +6,10 @@
                 "record": ["gpii.flowManager.local", "gpii.contextManager.app"],
                 "target": "{that flowManager}.options.gradeNames"
             },
+            "flowManager.resetAtStart": {
```

Comment:
I'm not sure we want this here either. It is really a behaviour which should be configured in at the gpii-app level rather than something which the system absolutely always does. I think best perhaps to put this into its own config and this can then be mixed in by whoever wants it.

Reply:
I've moved the config of "resetAtStart" into its own config file at [gpii/node_modules/flowManager/configs/gpii.flowManager.config.resetAtStart.base.json5](https://github.com/cindyli/universal/blob/GPII-3774/gpii/node_modules/flowManager/configs/gpii.flowManager.config.resetAtStart.base.json5). This file is not included in any other development or production config files at the moment so that gpii-app can explicitly include it when needed.